frontend/app/src/next/components/runs/run-id.tsx

```diff
@@ -46,7 +54,7 @@ export function RunId({ wfRun, taskRun, onClick }: RunIdProps) {
       <Tooltip>
         <TooltipTrigger asChild>
           <span>
-            {url && !onClick ? (
+            {url && !onClick && !noLink ? (
```

Comment:
nit: I usually prefer "positives" as opposed to double-negatives if possible (not a big deal)

Reply:
good catch -- meant to remove this anyway